improver_tests/standardise/test_StandardiseMetadata.py

```diff
@@ -292,6 +292,12 @@ def test_air_temperature_status_flag_coord_without_realization(self):
         self.assertArrayEqual(result.data, target.data)
         self.assertEqual(result.coords(), target.coords())
 
+    def test_long_name_removed(self):
+        cube = self.cube.copy()
+        cube.long_name = "kittens"
+        result = StandardiseMetadata().process(cube)
+        assert result.long_name is None, "long_name attribute removed or never added"
```

Comment:
I was not familiar with this assert syntax of appending a message to an error if one is raised. It's rather neat. In this case I am not sure that the message actually reflects what's happening if an error is raised, I imagine it's more something along the lines of:
`long_name removal expected, but long_name is not None`.

Am I misunderstanding the point of including this extra comment?

Reply:
Hi Ben, this is a good point, I have edited the message accordingly.